src/components/MetricsPane/SelectedSiteMetrics.jsx

```diff
@@ -58,7 +59,24 @@ export const SelectedSiteMetrics = ({
     updateCurrentSampleEvent,
   } = useContext(FilterProjectsContext)
   const [metricsView, setMetricsView] = useState(TAB_NAMES.summary)
-  const { site_notes: siteNotes, project_id: projectId, site_id: siteId } = selectedSampleEvent
+  const {
+    sample_event_id: sampleEventId,
+    site_notes: siteNotes,
+    project_id: projectId,
+    project_name: projectName,
+    site_id: siteId,
+    site_name: siteName,
+    project_admins: projectAdmins,
+    tags: organizations,
```

Comment:
tags = organizations? Good call to rename and clarify that! (also makes code below easier to read and understand now!)